### ISSUE TITLE
Cron jobs not running when created with kubectl

### ISSUE THREAD

**Issue by rbkaspr** [rancher/rancher#19355]
**What kind of request is this (question/bug/enhancement/feature request):** bug  

**Steps to reproduce (least amount of steps as possible):**
Create a cronjob via kubectl and a yaml spec  

**Result:**
Cronjob workloads created via kubectl do not trigger until the jobs trigger time is manually edited via the Rancher UI. After changing the cron specification through the UI, the job triggers normally, as expected  

**Other details that may be helpful:**

**Environment information**
Custom cluster (baremetal/on-premise)
Rancher version 2.2.0 (RKE HA Install)
Kubernetes v1.13.4
Docker version 18.09.3
```

**Comment by rbkaspr** [comment]
Can someone please take another look at this issue? We keep getting bitten by it with new CronJob deployments on our production environment, and it makes no sense for us to be able to deploy new CronJobs programmatically, but then have to manually go through the UI and make no-op edits just to get the jobs to trigger. This issue is still impacting us as of Rancher 2.2.2.

**Comment by vincent99** [comment]
This makes no sense unless the UI GET/PUT is accidentally fixing something wrong with your spec.  CronJobs are a native k8s resource that the Rancher backend isn't doing anything special with.

**Comment by rbkaspr** [comment]
I went ahead and revisited my CronJob yaml, and the issue seems to be with how I was defining arguments for the container. 

This notation is what I was previously using (and still use in my deployments):
```args: ["arg1", "arg2", "arg3"] ```
 
But that wasn't triggering properly. What seems to work properly is when I shift to this version: 
```
args: 
- arg1 
- arg2 
- arg3
```

This is potentially something worth digging into further with kubernetes itself, but is clearly not a Rancher issue. Closing.
